examples/src/fibonacci/fib2/prover.rs

```diff
@@ -3,6 +3,8 @@
 // This source code is licensed under the MIT license found in the
 // LICENSE file in the root directory of this source tree.
 
+use winterfell::DefaultConstraintEvaluator;
+
 use super::{
     BaseElement, DefaultRandomCoin, DefaultTraceLde, ElementHasher, FibAir, FieldElement,
     PhantomData, ProofOptions, Prover, Trace, TraceTable, TRACE_WIDTH,
```

Comment:
minor nit: could we import `DefaultConstraintEvaluator` from `super` as well (we might need to add it there)?